modules/ROOT/pages/fcct-config.adoc

```diff
@@ -1,53 +0,0 @@
-= Provisioning and Configuration
-
-This guide introduces the in-depth configuration specification for human-friendly FCOS Configuration (FCC) YAML files.
-
-TIP: Rather than read these detailed instructions, advanced users may want to reference the https://coreos.github.io/fcct/specs/[configuration specification] for an abbreviated list of options.
-
-== Ignition overview
-Ignition is a provisioning utility that reads a configuration file (in JSON format) and provisions an FCOS system based on that configuration. Configurable components include storage and filesystems, systemd units, and users.
-
-Ignition runs only once during the first boot of the system (in initramfs). Because Ignition runs so early in the boot process, it can re-partition disks, format filesystems, create users, and write files before the userspace begins to boot. As a result, systemd services are already written to disk when systemd starts, speeding the time to boot.
-
-=== Configuration process
-Ignition configurations are formatted as JSON, which is quick and easy for a machine to read. However, these files are not easy for humans to read or write. The solution is a two-step configuration process that is friendly for both humans and machines:
-
-. Produce a YAML-formatted Fedora CoreOS Configuration (FCC) file.
-. Run the Fedora CoreOS Configuration Transpiler (`fcct`) to convert the YAML file into a JSON Ignition file.
-
-During the transpilation process, `fcct` verifies the syntax of the YAML file, which can catch errors before you use it to launch the FCOS system.
-
-Once you have an Ignition (`.ign`) file, you can use it to boot an FCOS system in a VM or install it on bare metal.
```

Comment:
This is good introductory language.  Maybe it should move into `producing-ign`?

Reply:
moved.